rasa/cli/tutorial_project/data/flows.yml

```
@@ -0,0 +1,12 @@
+flows:
+  transfer_money:
+    description: This flow lets users send money to friends and family.
+    steps:
+      - id: "ask_recipient"
```

OPT: not sure if we also want to name this collect now that it isn't a question anymore
```suggestion
      - id: "collect_recipient"
```

---

IMO "ask" is still easier for a human to read (plus it's still in the utter key)